README.rst

```diff
@@ -22,26 +22,34 @@ target specific versions of python. This allows us to build Salt with
 the same version of Python accross MANY platforms, which allows for
 more rapid development and platform consistency.
 
-Usage
+Setup
 =====
 
-Install pop-build:
+### Clone the repo
+
+* git clone https://github.com/saltstack/salt-bin.git
+* cd salt-bin
+
+### Install dependencies
+
+* pip install -r requirements.txt
```

Comment:
This should just be changed to 
pip install pop-build

As the requirements file is for Salt itself, not the build process

Reply:
Would you be opposed to having two requirements.txt files
`salt_bin.requirements.txt` for dependencies required to run salt-bin builder
`salt.requirements.txt` for dependencies salt-bin builder uses?

Reply:
I think that the right approach would be a refactor, the requirements should all be in the conf directory so we can distribute salt-bin as a python project.
Let me get this :)